src/lib/index.js

```diff
@@ -1267,3 +1279,28 @@ lib.join2 = function(arr, mainSeparator, lastSeparator) {
 lib.bigFont = function(size) {
     return Math.round(1.2 * size);
 };
+
+/**
+ * Return the mouse position from the last event registered by D3.
+ * @returns An array with two numbers, representing the x and y coordinates of the mouse pointer
+ *   at the event relative to the targeted node.
+ */
+lib.getPositionFromD3Event = function() {
+    var firefoxVersion = lib.getFirefoxVersion();
```

Comment:
We don't want to call `getFirefoxVersion` every time this function is called.
So please declare `firefoxVersion` above `lib.getPositionFromD3Event`.

Reply:
Let's declare `isProblematicFirefox` outside the function.

Reply:
good idea, done